### PR TITLE
Solves issues

### PR DESCRIPTION
**Pull request by Igormandello**
This PR solves the issues #1 and #2

### CODE REVIEW

**Comment by serramatutu** [comment]
:+1: thanks. very much appreciated :)

**Comment by lucasdinnouti** [comment]
That's exactly what I was expecting. Much thanks

File: js/jsTradutor.js [hunk 1]
```diff
@@ -82,183 +85,60 @@ function caesar(mensagem, chave)
...
-          {
-              indiceDaLetraAtual = alf.indexOf(input.substring(i,(i+1)));
-              outText += alf[(indiceDaLetraAtual + (chave * sinal)) % alf.length];
-              i++;
-          }
-          else{
-            if(input.substring(i,(i+1)) == " ")
-            {
-                outText += input.substring(i,(i+1));
+    //TURISTADAAAAAAAAAAAAAAA OUT
```
Comment (gabrielpallotta) [inline js/jsTradutor.js:95]: I think this comment is unnecessary here. Maybe you should remove it?

**Review by gabrielpallotta — CHANGES_REQUESTED** [review]

File: js/jsTradutor.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+}
+
 
 function criptografar(mensagem, idCripto, c)
 {
     //Declarando o modal
     var modal = document.getElementById("mTxt");
 
-    mensagem = mensagem.toUpperCase();
+    mensagem = mensagem;
```
Comment (gabrielpallotta) [inline js/jsTradutor.js:13]: This does not seem correct. Please fix this!!!!!!
```suggestion
```

**Review by gabrielpallotta — CHANGES_REQUESTED** [review]

**Pull request merged**